tests/functional/aws-node-sdk/test/bucket/getCors.js

```diff
@@ -0,0 +1,75 @@
+import assert from 'assert';
+
+import withV4 from '../support/withV4';
+import BucketUtility from '../../lib/utility/bucket-util';
+
+const bucketName = 'testgetcorsbucket';
+const sampleCors = { CORSRules: [
+  { AllowedMethods: ['PUT', 'POST', 'DELETE'],
+    AllowedOrigins: ['http://www.example.com'],
+    AllowedHeaders: ['*'],
+    MaxAgeSeconds: 3000,
+    ExposeHeaders: ['x-amz-server-side-encryption'] },
+  { AllowedMethods: ['GET'],
+    AllowedOrigins: ['*'],
+    AllowedHeaders: ['*'],
+    MaxAgeSeconds: 3000 },
+] };
+
+describe('GET bucket cors', () => {
+    withV4(sigCfg => {
+        const bucketUtil = new BucketUtility('default', sigCfg);
+        const s3 = bucketUtil.s3;
+
+        afterEach(() => bucketUtil.deleteOne(bucketName));
+
+        describe('on bucket with existing cors configuration', () => {
+            before(() =>
+                s3.createBucketAsync({ Bucket: bucketName })
+                .then(() => s3.putBucketCorsAsync({
+                    Bucket: bucketName,
+                    CORSConfiguration: sampleCors,
+                })));
+
+            it('should return cors configuration successfully', done => {
+                s3.getBucketCors({ Bucket: bucketName }, (err, data) => {
+                    assert.strictEqual(err, null,
+                        `Found unexpected err ${err}`);
+                    assert.deepStrictEqual(data.CORSRules[0],
```

Comment:
I tried to remember why the output is different. AWS Node SDK returns an empty array for an unset attribute like ExposeHeaders. I added an empty array to the original CorsConfiguration object to make it easier to compare without comparing the individual values.